README.md

```diff
@@ -4,4 +4,9 @@ Module 1:creating Hello Word Exercise
 .A .css file that includes one class with styles for your header (make it in your favorite color).
 .A test.md file with the text "This file should be ignored by git".
 .Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
-.Open a pull request
\ No newline at end of file
+.Open a pull request
+
```

Comment:
- [ ] Your `.gitignore` file should also ignore the `node_modules` directory as committing this directory makes your Pull Request unreadable. Right now it only ignores the `test.md` file
![node_m](https://user-images.githubusercontent.com/36542986/133323513-3cd550f9-60f0-45b8-92dd-e6e10668a0e5.jpg)